strings/salesman_replacement.json

```diff
@@ -0,0 +1,49 @@
+{
+
+    "salesman": {
+        "shuttle": "cungadero",
+        "clam": "pipis",
+        "clams": "pipis",
+        "egg": "pipis",
+        "eggs": "pipis",
+        "love": "hyperlink blocked",
+        "captain": "big shot",
+        "dollar": "kromer",
+        "dollars": "kromer",
+        "credit": "kromer",
+        "credits": "kromer",
+        "basement": "deep abyss",
```

Comment:
this is really uncommon terminology

Reply:
Its for consistency with the game's interpretation. Just a funny thing for when people wanna talk about lower z-levels on multi-z stations i guess.

Reply:
make "bottom level" turn into "deep abyss"?

Reply:
credit/credits are used decently in game, but i'd suggest adding "maintenance" and "maint" to the list of things replaced with "deep abyss"